src/client/voice/networking/VoiceUDPClient.js

```diff
@@ -90,46 +90,46 @@ class VoiceConnectionUDPClient extends EventEmitter {
 
   async createUDPSocket(address) {
     this.discordAddress = address;
-    const socket = (this.socket = udp.createSocket('udp4'));
-    socket.on('error', e => {
-      this.emit('debug', `[UDP] Error: ${e}`);
-      this.emit('error', e);
-    });
-    socket.on('close', () => {
-      this.emit('debug', '[UDP] socket closed');
-    });
-    this.emit('debug', `[UDP] created socket`);
-    socket.once('message', message => {
-      this.emit('debug', `[UDP] message: [${[...message]}] (${message})`);
-      // Stop if the sockets have been deleted because the connection has been closed already
-      if (!this.voiceConnection.sockets.ws) return;
-
-      const packet = parseLocalPacket(message);
-      if (packet.error) {
-        this.emit('debug', `[UDP] ERROR: ${packet.error}`);
-        this.emit('error', packet.error);
-        return;
-      }
+    const socket = (this.socket = udp.createSocket('udp4'))
+      .on('error', e => {
```

Comment:
Honestly, I'm not a fan of inlinig these with the assigning/creation of the socket... Maybe make it `socket.on().on()` afterwards